prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -201,8 +201,8 @@ int rxr_pkt_init_ctrl(struct rxr_ep *rxr_ep, int entry_type, void *x_entry,
 		ret = rxr_pkt_init_dc_write_rta(rxr_ep, (struct rxr_tx_entry *)x_entry, pkt_entry);
 		break;
 	default:
-		ret = -FI_EINVAL;
 		assert(0 && "unknown pkt type to init");
+		ret = -FI_EINVAL;
```

Comment:
what was the warning before?

Reply:
I don't remember anymore. It was probably something along the lines of "ret is unused" or "setting ret before an assert is not useful"

Reply:
Alright. If you find it by any chance, pls leave it here.
In addition, did you miss some changes in the most recent push. I dont think it's just one line change in this PR?

Reply:
Yeah I messed things up, fixing

Reply:
Ended up pushing the wrong branch, should be fine now